dhis2/capture-app
# feat: [DHIS2-13299][DHIS2-16291] Related Stages

TECH-summary:
This widget checks if there are a relationship type of program stage to program stage. If there are on, there is a widget appearing underneath the New Enrollment event form.

How to test:
1. Create a new relationship type
2. This type should be `Event in program or program stage` => `Event in program or program stage` in the same program
3.  Make sure to check the sharing setting so that the user can capture data

<img width="1727" alt="image" src="https://github.com/dhis2/capture-app/assets/12842868/04c8727d-1eab-4f30-a82b-4a98861d89af">

**This PR is made up of several issues who have all been reviewed. Please only check the latest commit**
